app/pulp/app/db-reset.sh

```diff
@@ -1,9 +1,15 @@
-#!/bin/bash -x
-
-# This assumes you're running in vagrant.
-echo "drop database pulp; create database pulp owner pulp" | sudo -u postgres psql
-
-rm -rf migrations
```

Comment:
It's rude. :)

I want to be able to reset my DB without blowing away all my migrations, and "manage.py reset_db --noinput" doesn't do it. So it's either this, or rename the script "db-reset-and-oh-yeah-im-deleting-your-migrations-too.sh". There's also the forthcoming problem of the plugins having their own migrations to destroy, which should probably be handled by a more sophisticated script that we've yet to invent.